noop/index.js

```diff
@@ -0,0 +1,43 @@
+const {default: traverse} = require("@babel/traverse");
+const {looksLike} = require("../utils/looks-like");
+
+
+module.exports = () => {
+	return {
+		name: "willMutate",
+		visitor: {
+			/**
+             * We need to traverse the "Program" so we can know if we need to inject
+             * the Proxy handler or not.
```

Comment:
Is this comment accurately describing this program?

Reply:
Nooooope lol. Copy + paste hits again